builder/vsphere/common/step_connect.go

```diff
@@ -71,9 +72,9 @@ func (s *StepConnect) Cleanup(state multistep.StateBag) {
 	driver := state.Get("driver").(driver.Driver)
 	errorRestClient, errorSoapClient := driver.Cleanup()
 	if errorRestClient != nil {
-		ui.Message("Error closing rest client session: " + errorRestClient.Error())
+		log.Printf("[WARN] Couldn't close rest client session. Could be already closed: %s", errorRestClient.Error())
 	}
 	if errorSoapClient != nil {
-		ui.Message("Error closing soap client session: " + errorRestClient.Error())
+		log.Printf("[WARN] Couldn't close rest client session. %s", errorRestClient.Error())
```

Comment:
I believe this is a typo. 
```suggestion
		log.Printf("[WARN] Couldn't close rest client session. %s", errorSoapClient.Error())
```

Reply:
I think there are two typos present here: 

```suggestion
		log.Printf("[WARN] Couldn't close soap client session. %s", errorSoapClient.Error())
```